capstone-ui/src/Components/App/App.jsx

```diff
@@ -71,12 +71,15 @@ export default function App() {
             </div>
             <div className="content">
               <Routes>
-                <Route path='/' element={ user ? (<div className='loggedin-home'>
-                                                    <Feed loggedin={true}/> 
-                                                    <Recommendations /> 
-                                                  </div>) : (
-                                                    <Feed loggedin={false}/>
-                                                  )}/>
+                <Route path='/' element={<div className={`${user ? "loggedin-home" : "" }`}>
```

Comment:
Let's avoid adding too much nested components here. Ideally, we would want Just one component. You can create a Home component and move these components / logic there.